assets/src/styles/layouts/_header.scss

```diff
@@ -141,8 +156,6 @@ body:not(.home) {
 
 .reading-header__title {
   background-color: var(--footer-bg);
-  border-top: solid 2px var(--body-bg);
```

Comment:
remove border so that title element appears to have same dimesions as blocks on either side